default.nix

```diff
@@ -5,32 +5,42 @@ with lib;
 
 rec {
   qemu = pkgs.qemu.override {
+    guestAgentSupport = false;
+    numaSupport = false;
+    seccompSupport = false;
+    alsaSupport = false;
+    pulseSupport = false;
+    jackSupport = false;
     sdlSupport = false;
+    gtkSupport = false;
     vncSupport = false;
+    smartcardSupport = false;
     spiceSupport = false;
-    pulseSupport = false;
-    smbdSupport = true;
+    ncursesSupport = false;
+    libiscsiSupport = false;
+    tpmSupport = false;
+
     hostCpuOnly = true;
   };
 
-  baseKernelPackages = linuxPackages_4_14;
+  baseKernelPackages = linuxPackages; # tested up to 5.19
 
   kconfig = kernelConfig.override {
     linux = baseKernelPackages.kernel;
   } {
     config = {
       X86 = true;
       "64BIT" = true;
-      #"PRINTK" "BUG" # extra debugging
+      # PRINTK = true; # extra debugging
 
-      LOCAL_VERSION = "qeval";
```

Comment:
This should be `LOCALVERSION`, but adding it causes the module directory to not match.